etc/skydive.yml.default

```diff
@@ -65,7 +65,7 @@ analyzer:
 
   # Flow storage engine
   flow:
-    # Storage backend name: myelasticsearch, myorientdb
+    # Storage backend name: myelasticsearch, myorientdb, myobjectstorage
```

Comment:
Thanks for your contribution
Could you describe your main motivation of using aws storage instead of elasticsearch (for example) ?

Reply:
The main advantage of object storage is its price, so it is great for storing large amounts of data (say you want to keep an archive of years full of flow logs).
See Amazon VPC flow logs as example:
https://docs.aws.amazon.com/vpc/latest/userguide/flow-logs.html